frontend/src/components/navbar/top-navbar.tsx

```diff
@@ -11,48 +10,132 @@ interface TopNavBarProps {
     className?: string
 }
 
+const menuToggleStyles = {
+    width: 150,
+    height: 50,
+    opacity: 0.7,
+    cursor: 'pointer',
+    whiteSpace: 'nowrap' as const,
+    userSelect: 'none' as const,
+};
+
+const Wrapper = styled.div({
+    position: 'relative',
+    display: 'inline-block',
+});
+
 export const StyledLink = styled(NavLink)({
     textDecoration: 'none',
-    color: colors.text,
+    color: 'white',
+    fontFamily: 'System-ui',
+    fontSize: '18px',
+    fontWeight: 700,
+    lineHeight: '28px',
+    textAlign: 'left',
+});
+
+const NavbarStyledLink = styled(NavLink)({
```

Comment:
Can use flex property here instead of introducing a new Flex Container. Also Chris mentioned that we are moving away from styled components, so this can be made into a FC.